### ISSUE TITLE
console log data not displayed in order

### ISSUE THREAD

**Issue by cdrainxv** [cdrainxv/xkcli#1]
Currently the order of the terminal output is:
1. title
2. alt text
3. image

However, the image and the alt text order should be switched as defined here:
```js
  console.log(chalk.bold.underline.blueBright(title))
  imgcat(img, { log: true })
  console.log(chalk.greenBright(alt))
```

There should be a way to pause the second console.log until after the the image is fully loaded.
